manifests/t/TexasInstruments/TIConnectCE/6.0.1.344/TexasInstruments.TIConnectCE.yaml

```
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: TexasInstruments.TIConnectCE

```

```suggestion
PackageIdentifier: TexasInstruments.TIConnect.CE
```
After updating this, you will need to rename the files and place them in a new folder